package.xml

```diff
@@ -11,8 +11,15 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend>rapidjson-dev</build_depend>  
-  <build_depend>libgz-cmake3-dev</build_depend>
-  <build_depend>libgz-sim7-dev</build_depend>
+
+  <!-- Harmonic -->
+  <depend condition="$GZ_VERSION == harmonic">gz-cmake3</depend>
+  <depend condition="$GZ_VERSION == harmonic">gz-sim8</depend>
```

Comment:
Is this known to not work?
```
rryan@B650-970:~/Dev/ros2_ws/src/ardupilot_gazebo$ rosdep install --from-paths . --ignore-src
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
ardupilot_gazebo: Cannot locate rosdep definition for [gz-sim8]
```

Reply:
Taking my steer from: https://github.com/gazebosim/ros_gz/blob/a982161db38058f3c3b6b69f660184930484f077/ros_gz_sim/package.xml#L28-L32
which I assumed correct as it is OSRF maintained.

Reply:
Hopefully they fix it at some point.